### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by dmikurube**
@muga And releasing v0.2.0 finally... Some concerns about Gradle (Maven) package group, and Bintray organization. Where are good to release this do you think?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,8 +10,8 @@ plugins {
 }
 import com.github.jrubygradle.JRubyExec
 
-group = 'org.embulk.base.restclient'
-version = '0.1.3'
+group = 'org.embulk'
```
Comment (dmikurube) [inline build.gradle]: TBD: Do we have this directly under `org.embulk` or in an independent group?

File: build.gradle [hunk 2]
```diff
@@ -63,10 +63,10 @@ bintray {
     publish = true
 
     pkg {
-        userOrg = 'embulk-base-restclient'
+        userOrg = 'embulk'
```
Comment (dmikurube) [inline build.gradle]: TBD: Same above for Bintray.

File: build.gradle [hunk 3]
```diff
@@ -63,10 +63,10 @@ bintray {
...
         repo = 'maven'
         name = project.name
-        desc = 'Base classes for Embulk plugins working as REST clients'
+        desc = 'Base class library to access RESTful services'
         websiteUrl = 'https://github.com/embulk/embulk-base-restclient'
```
Comment (dmikurube) [inline build.gradle]: To move into github.com/embulk.

**Comment by dmikurube** [comment]
Chatted. To merge!

**Comment by muga** [comment]
As summary, 

group: org.embulk.base.restclient
artifact: embulk-base-restclient
Bintray org: embulk-base-restclient
the URL: https://bintray.com/embulk-base-restclient/maven/embulk-base-restclient

**Pull request merged**